packages/auth/src/grant/model.ts

```diff
@@ -39,8 +40,8 @@ export class Grant extends BaseModel {
         to: 'accesses.grantId'
       }
     }
-  }
-
+  })
+  public access!: Access[]
```

Comment:
Is this field intentional? I don't see a migration for it in the grants table, but I believe the `Access` table already has a foreign key in the form of the grant id as well.

Reply:
It is because otherwise I can't get the access property [here](https://github.com/interledger/rafiki/blob/8b4e7859b94991a5a187e30c47c5474cefdfb84d/packages/auth/src/accessToken/routes.ts#L55). Like you said, we do have the `grantId` as foreign key in the access table. Due to the `relationsMapping`, we can easily add `access` as a property to the grant without having to store it in the table.